src/cli.js

```diff
@@ -218,17 +219,33 @@ async function runCmd (argv, stdout, stderr) {
       );
       if (existsSync(outDir))
         rimraf.sync(outDir);
+      mkdirp.sync(outDir);
       run = true;
 
     // fallthrough
     case "build":
       if (args._.length > 2)
         errTooManyArguments("build");
 
+      function hasTypeModule (path) {
+        let root = resolve('/');
+        while ((path = resolve(path, '..')) !== root) {
+          try {
+            return JSON.parse(readFileSync(eval('resolve')(path, 'package.json')).toString()).type === 'module';
+          }
+          catch (e) {
+            if (e.code === 'ENOENT')
+              continue;
```

Comment:
Should this be `break` instead of `continue`? I would assume `ENOENT` means that walking up the tree to the root would still yield `ENOENT` but not sure how Node.js handles that case.

Reply:
This is how the Node.js package boundary lookup works. See https://nodejs.org/dist/latest-v16.x/docs/api/packages.html#packages_package_json_and_file_extensions.

Reply:
Oh I was thinking of `ENOENT` in the case it means "directory does not exist" but that wouldn't happen since package.json is a file, not a directory 😌